### DISCUSSION TITLE
Moving a package between organisations

### DISCUSSION THREAD

**Discussion by Skogsfrae** [onepub-dev/onepub#199] [Q&A] [answered]
Is it possible to move a package from one organisation to another?
For example moving a package from an organisation created for a user with a free plan to an organisation created for a team with a pro plan and thus changing the package ownership?

**Comment by bsutton** [comment, marked as answer]
Hmm, you keep asking the hard questions:)

So the answer is no but I don't think you need to.

You can simply publish the same package to the new organisation.

The dependency url includes the organisation id so you can have the same
package in multiple organisations.

You can also belong to multiple organisations.

https://onepub.dev/faq/73a8b29d-ab76-4cb8-98e5-bacafb338b6a

To change a dependency to the new organisation simply login to the
alternate organisation from the cli.

The run
onepub pub add xxxx
Again and the dependency will be updated to the new organisation.

**Reply by Skogsfrae** [reply]
Sorry, I don't mean to be annoying 😂
I thought it would be useful to ask here in case other people had the same doubts :)

Thanks for your quick answers ;)

**Comment by bsutton** [comment]
Keep asking questions.

We are in beta to get feedback so your questions are great.

Can I ask for your overall opinion?
Any obvious negatives?

How do the price points sit?

**Reply by Skogsfrae** [reply]
I just started using it, but overall it looks pretty good so far.
The price seems very good to me and to be honest I've looked for a solution like onepub for almost a year.
I'm very very happy right now
